scripts/src/release/releasechecker.py

```diff
@@ -62,10 +71,39 @@ def check_if_only_charts_are_included(api_url):
             file_path = f["filename"]
             match = chart_pattern.match(file_path)
             if not match:
+                print(f"[INFO non chart file found: {file_path}")
                 return False
 
     return True
 
+def check_if_no_charts_are_included(api_url):
+
+    print("[INFO] check if PR contains any chart files")
+    files_api_url = f'{api_url}/files'
+    headers = {'Accept': 'application/vnd.github.v3+json'}
+    chart_pattern = re.compile(r"charts/"+TYPE_MATCH_EXPRESSION+"/([\w-]+)/([\w-]+)/.*")
+    page_number = 1
+    max_page_size,page_size = 100,100
+    file_count = 0
+
+    while page_size == max_page_size:
```

Comment:
Em we've used this pattern multiple times, wondering if we could move it to the util file.

Reply:
LGTM, please restart the testing